CPS-?/README.md

```diff
@@ -0,0 +1,170 @@
+
+These recommendations are the result of a collaborative effort among cybersecurity, financial, and game theory experts, as well as participants of the Voltaire workshop, including developers, stake pool operators, representatives from IOG, Cardano Foundation, and Emurgo, members of Project Catalyst, and members of the Cardano community.
+
+
+## Problem
+
+The Cardano community's on-chain governance mechanism, Voltaire, is susceptible to various types of attacks that may compromise its security. This system defines on-chain decision-making that is automatically executed when a certain threshold of approval by human actors is met. To ensure maximum awareness of potential threats and vulnerabilities, this document will enumerate and describe each threat or vulnerability, including recommended mitigation techniques. It is crucial that all stakeholders, including voters, developers, and governance actors, are informed of these risks to identify and prevent any damage or, in the case of a threat event, facilitate recovery.
+
+In rare instances, a threat may be identified that requires responsible reporting procedures to prevent exposing an active vulnerability to bad actors seeking to exploit it. In such cases, it is essential to notify the appropriate group or entity with the ressources, or are responsible for mitigating the threat in a confidential manner.
+
+### #1: Unknown software vulnerability affecting governance is discovered.
+
+An assumption is that are no known software back doors. Unknown software vulnerabilities may be discovered.
+
+Recommendation: The person discovering the vulnerability should document the circumstances and details in which the vulnerability is present and how to reproduce the vulnerability. Then report the vulnerability in a confidential manner using a secure method to the developer team with the resources to correct the vulnerability.
+
+### #2: Emergency group to support the disaster recovery plan.
+  -attacks on persons if known
+  -we don’t want attackers know who that group is?
+
+Recommendation:
+
+### #3: Emergency change that needs to be done quickly.
+  -a nuclear option
+  -change requires “just do it” 
+  -a security that fix must be implemented but CANNOT be communicated publicly
+
+Recommendation:
+
+### #4: Operational Security (OPSEC) of the committee.
+  -physical security
+  -digital security
+
+Recommendation:
+
+### #5: Committee keys lost or stolen or committee keys sold to a bad actor.
+
+Recommendation:
+
```

Comment:
- Lost/Stolen : Member must be able to sign with the same wallet/stake key to authenticate for burn/mint of previous key

- Sold to bad actor : Loss of collateral / Keys destroyed by committee vote

Reply:
I think the result of lost, stolen, or sold is the same. The keys must be burned and new ones generated. I could be wrong.